vowpalwabbit/example.cc

```diff
@@ -314,19 +314,18 @@ example* alloc_examples(size_t, size_t count)
 {
   example* ec = calloc_or_throw<example>(count);
   if (ec == nullptr) return nullptr;
-  for (size_t i = 0; i < count; i++)
-  {
-    ec[i].ft_offset = 0;
-  }
+  for (size_t i = 0; i < count; i++) { new (ec + i) example; }
```

Comment:
this doesn't need to be `new (ec + i) example()`?

Reply:
I think the parentheses can be omitted when there are no args